tracer/build/artifacts/dd-dotnet.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Get the directory of the script
+DIR=$(dirname "$0")
+
+
+# Read the /etc/os-release to detect distribution
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+else
+    echo "Unable to find /etc/os-release. Cannot determine the distribution."
+    exit 1
+fi
+
+# Check the machine architecture
+ARCH="$(uname -m)"
+
+DD_DOTNET_PATH=""
+
+# Set the DD_DOTNET_PATH according to the distribution and architecture
+if [ "$ID" = "alpine" ]; then
+    if [ "$ARCH" = "x86_64" ]; then
+        DD_DOTNET_PATH="$DIR/linux-musl-x64/dd-dotnet"
+    elif [ "$ARCH" = "aarch64" ]; then
+        echo "Alpine ARM64 is not supported."
+        exit 1
+    else
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+    fi
+else
+    if [ "$ARCH" = "x86_64" ]; then
+        DD_DOTNET_PATH="$DIR/linux-x64/dd-dotnet"
+    elif [ "$ARCH" = "aarch64" ]; then
+        DD_DOTNET_PATH="$DIR/linux-arm64/dd-dotnet"
+    else
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+    fi
+fi
+
+# Check if DD_DOTNET_PATH is set and the file exists
+if [ -z "$DD_DOTNET_PATH" ]; then
+    echo "Error determining dd-dotnet path."
```

Comment:
This line can't be hit right? It's fine keeping it if so, but maybe a comment for us?